dev/ci/ci-hott.sh

```diff
@@ -7,4 +7,4 @@ HoTT_CI_DIR=${CI_BUILD_DIR}/HoTT
 
 git_checkout ${HoTT_CI_BRANCH} ${HoTT_CI_GITURL} ${HoTT_CI_DIR}
 
-( cd ${HoTT_CI_DIR} && ./autogen.sh && ./configure && make -j ${NJOBS} )
+( cd ${HoTT_CI_DIR} && ./autogen.sh && ./configure && ./etc/coq-scripts/timing/make-pretty-timed-or-error.sh -j ${NJOBS} && make validate )
```

Comment:
I meant the timing script, I am not sure we want that, and in that form.

I we go that route, I think the timing script should be standardized and merged into Coq, I am not comfortable relying on possibly out-of-sync scripts among Travis developments.